src/recently-viewed.css

```
@@ -0,0 +1,263 @@
+.recently-viewed-card::after {
+  content: "";
+  position: absolute;
+  inset: 0;
+  border: 1px solid #f4f4f5;
+  border-radius: inherit;
+  pointer-events: none;
+  z-index: 1;
+}
+
+.recently-viewed-card:hover {
+  box-shadow: 0 20px 25px -5px rgba(15, 23, 42, 0.08), 0 8px 10px -6px rgba(15, 23, 42, 0.08);
```

Stylelint (17.11.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)